runtime/doc/eval.txt

```diff
@@ -7309,12 +7311,24 @@ matchend({expr}, {pat} [, {start} [, {count}]])			*matchend()*
 			GetText()->matchend('word')
 
 
-matchfuzzy({list}, {str})			*matchfuzzy()*
-		Returns a list with all the strings in {list} that fuzzy
-		match {str}. The strings in the returned list are sorted
-		based on the matching score. {str} is treated as a literal
-		string and regular expression matching is NOT supported.
-		The maximum supported {str} length is 256.
+matchfuzzy({list}, {str} [, {key}])
```

Comment:
based on my comment here might be good to not support key but instead support expression with `v:val` which can be optimized to extract key. https://github.com/vim/vim/pull/6932#issuecomment-691677822